### PR TITLE
[jsk_tools] Correct order of ROS_IP in list of hostname -I

### LINKED ISSUE #1170: [bug] [jsk_tools] rossetip does not set correct ip on my laptop
```
% source 99.jsk_tools.zsh
[http://baxter:11311][240d:1a:2a:300:7a31:c1ff:febb:6b6c] wkentaro at cobra
… in ~/…/src/jsk-ros-pkg/jsk_common/jsk_tools/env-hooks on master* rosp jsk_tools 
% rossetip
set ROS_IP and ROS_HOSTNAME to 240d:1a:2a:300:7a31:c1ff:febb:6b6c
[http://baxter:11311][240d:1a:2a:300:7a31:c1ff:febb:6b6c] wkentaro at cobra
… in ~/…/src/jsk-ros-pkg/jsk_common/jsk_tools/env-hooks on master* rosp jsk_tools 
% git checkout 3ff082afa25751eadfb0ec391203c2bb446ff2c7
Note: checking out '3ff082afa25751eadfb0ec391203c2bb446ff2c7'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b new_branch_name

HEAD is now at 3ff082a... [jsk_tools/99.jsk_tools.sh] fix typo
[http://baxter:11311][240d:1a:2a:300:7a31:c1ff:febb:6b6c] wkentaro at cobra
… in ~/…/src/jsk-ros-pkg/jsk_common/jsk_tools/env-hooks on 3ff082a... rosp jsk_tools 
% git lg
[http://baxter:11311][240d:1a:2a:300:7a31:c1ff:febb:6b6c] wkentaro at cobra
… in ~/…/src/jsk-ros-pkg/jsk_common/jsk_tools/env-hooks on 3ff082a... rosp jsk_tools 
% source 99.jsk_tools.zsh
[http://baxter:11311][240d:1a:2a:300:7a31:c1ff:febb:6b6c] wkentaro at cobra
… in ~/…/src/jsk-ros-pkg/jsk_common/jsk_tools/env-hooks on 3ff082a... rosp jsk_tools 
% rossetip
set ROS_IP and ROS_HOSTNAME to 192.168.1.4
[http://baxter:11311][192.168.1.4] wkentaro at cobra
… in ~/…/src/jsk-ros-pkg/jsk_common/jsk_tools/env-hooks on 3ff082a... rosp jsk_tools 
```

This may happens after the change on ee1c10a . (3ff082afa25751eadfb0ec391203c2bb446ff2c7 is one commit behind of ee1c10a)

### PR DESCRIPTION
**Pull request by wkentaro**
Closes #1170

### CODE REVIEW

**Comment by wkentaro** [comment]
@furushchev 
Please review if this change is fine.

**Comment by furushchev** [comment]
Rather it seems to be better to remove `elif` part (https://github.com/wkentaro/jsk_common/blob/correct-order-ros-ip/jsk_tools/env-hooks/99.jsk_tools.sh#L98-L100) than to add `break`

File: jsk_tools/env-hooks/99.jsk_tools.sh [hunk 1]
```diff
@@ -95,9 +95,7 @@ rossetip_addr() {
         if echo $ip | egrep "^172.17.42.|^127.0." >/dev/null; then
             # skip docker/local host
             continue
-        elif [ "${mask_targetip}" = "" ]; then
-            export ROS_IP=$ip
-        elif echo $ip | grep $mask_target_ip; then
+        elif echo $ip | grep "$mask_target_ip" >/dev/null; then
```
Comment (wkentaro) [inline jsk_tools/env-hooks/99.jsk_tools.sh]: quotes should be added for blank `mask_target_ip`.
and '>/dev/null' to remove stdout.

**Comment by wkentaro** [comment]
@furushchev 
Please review again.

**Comment by furushchev** [comment]
@wkentaro :+1:

**Comment by wkentaro** [comment]
@garaemon Please merge.

**Pull request merged**